ProcessMaker/processmaker
# FOUR-7269: Remove active user notifications query from view

## Issue & Reproduction Steps
If you look in the [base view (the view which is extended by virtually all other views) on line 44](https://github.com/ProcessMaker/processmaker/blob/f69f34fd2009044efdad6d6bcbc9a452f54d2b6c/resources/views/layouts/layout.blade.php#L44), you’ll see all active notifications are queried for the currently authenticated user and are then converted to JSON.  Since an API call already exists to populate this property, the query happening in the view is duplicative and slows down response time (usually only marginally).

## Solution
There already is an API call which wrote over the window.ProcessMaker.messages property on page load anyway, so replacing the query with an empty array will prevent the duplicative query from happening.

## How to Test
You should be able to load any dashboard page and have the most recent notifications load without any errors. You should also use the command line and run a curl request to a dashboard page to verify the `window.ProcessMaker.messages` property is an empty array.

## Related Tickets & Packages
- [FOUR-7269](https://processmaker.atlassian.net/browse/FOUR-7269)
- [package-webentry PR](https://github.com/ProcessMaker/package-webentry/pull/181)

## Code Review Checklist


[FOUR-7269]: https://processmaker.atlassian.net/browse/FOUR-7269?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

.
